bitnami/zookeeper/README.md

```diff
@@ -111,7 +111,7 @@ The command removes all the Kubernetes components associated with the chart and
 
 | Name                                 | Description                                                                                                                                                                                       | Value           |
 | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- |
-| `extraVolumes`                       | Extra volumes                                                                                                                                                                                     | `[]`            |
+| `extraVolumes`                       | Extra volumes.                                                                                                                                                                                     | `[]`            |
```

Comment:
this point is not necessary. Please delete it. Please, leave the line as the original one (no changes, please).